doc/source/learning/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _learningmaterials:
+
```

Comment:
assuming we include this page on the contributor's guide, I guess we could simplify this page to directly include "How to learn what's necessary to be able to contribute to socok8s".

The way I see it, the first item could be to know our practices, which is the content of contributor.rst.
The next would be to point the user to the deployment guide.
Then, k8s concepts to understand what the user just did. (learning/k8s.rst seems good on that point), and point to caasp docs.
Then, how to contribute to OSH, and Airship, pointing to the upstream docs.
Finally, pointing to our practices (sprint/backlogs urls)

But I trust you on presenting this information the way it would suit a new member best.

Reply:
We changed a bit the suggested order. Now you can find the learning materials at the beginning of the Developer Documentation section. This way you can read about the high level picture before entering the low level details.